src/solvers/dg_tree/dg.jl

```diff
@@ -132,17 +139,74 @@ end
 
 
 
+abstract type AbstractSurfaceIntegral end
+
+"""
+    SurfaceIntegralWeakForm
```

Comment:
```suggestion
    SurfaceIntegralWeakForm(surface_flux)
```
?

Also, for symmetry, should we use `surface_flux=flux_central` as the default, such that if you just specify `SurfaceIntegralWeakForm()`, `VolumeIntegralWeakForm()` you get all `flux_central`?

Reply:
Implemented in 157ece51.